packages/identity/tests/index.test.ts

```diff
@@ -2,20 +2,20 @@ import { Identity } from "../src"
 
 describe("Identity", () => {
     const privateKeyText = "secret"
-    const privateKeyHexadecimal = "dd998334940df8931b76d899fdb189415f7ff4280599f03a7574725a166aad7d"
+    const privateKeyBuffer = Buffer.from(privateKeyText)
```

Comment:
I think it would be good to test a byte input which isn't the same as the string input.